pkg/printer/printer.go

```diff
@@ -242,17 +260,22 @@ func (p *Printer) WriteProtoFlow(res *observerpb.GetFlowsResponse) error {
 			return fmt.Errorf("failed to write out packet: %v", err)
 		}
 	case CompactOutput:
+		var err error
```

Comment:
This is shadowing `err` declared at line 195. Although as it stand it is harmless, it is also useless.